### DISCUSSION TITLE
Are XTTS scripts now updated to fully support Hindi?

### DISCUSSION THREAD

**Discussion by StellarBeing25** [idiap/coqui-ai-TTS#73] [Q&A] [answered]
According to this https://github.com/erew123/alltalk_tts/issues/178#issuecomment-2064999354 Coqui has added Hindi language support but did not update all their scripts to support Hindi. Is it now updated in this fork?

**Comment by eginhard** [comment, marked as answer]
Yes, the support for Hindi was improved in #4 and #64 and everything should work fine now. ~~There's one PR remaining (https://github.com/idiap/coqui-ai-TTS/pull/64) for Hindi support in the tokenizer. I'll try to merge that soon, but you can already test that branch in the meantime.~~

**Reply by eginhard** [reply]
That PR is also merged now. Let me know in case you encounter any other issues.

**Comment by Iharsh-Mishra** [comment]
For hindi it is not able to convert numerical values like 1948.

**Reply by eginhard** [reply]
Yes, Coqui can handle only some basic text normalization. You'll need to convert it into words yourself first.
